packages/fxa-settings/src/components/Settings/Page2faReplaceRecoveryCodes/index.tsx

```diff
@@ -174,15 +174,19 @@ export const Page2faReplaceRecoveryCodes = (_: RouteComponentProps) => {
                 type="button"
                 className="cta-neutral cta-base-p mx-2 flex-1"
                 onClick={goHome}
```

Comment:
There's something about the navigation in this click handler that's tripping up the glean event. It's not emitted unless I comment out the click handler.